chart/flux/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: "1.15.0"
-version: 0.15.0
+version: 0.15.1
```

Chart releasing has its own pace

```suggestion
version: 0.15.0
```

---

@2opremio are you able to apply the suggestion, @adusumillipraveen is on my team and is on leave now till next year

---

Unfortunately I don't have permission to push to @adusumillipraveen 's repository

---

Actually, I was able to push from the command line, but GitHub didn't let me commit the suggestion.

---

I had an issue with that last week, applying a suggestion for someone else, and getting access denied. They then showed a screenshot showing the allow edits from maintainers was ticked =/

---

Yeah, that's what seems to be happening here. I think that the suggestion application doesn't work for PRs from forks.